### PR TITLE
Added some additional fail safes for transcript load timing.

### PR DESCRIPTION
**Pull request by nikkimk**

### CODE REVIEW

File: elements/a11y-media-player/src/a11y-media-player.js [hunk 1]
```diff
@@ -754,6 +754,7 @@ class A11yMediaPlayer extends SimpleColors {
     let root = this;
     super.connectedCallback();
     this.__loadedTracks = this.getloadedTracks();
+    console.log('loadedtracks',this.getloadedTracks());
```
Comment (btopro) [inline elements/a11y-media-player/src/a11y-media-player.js]: booo console.log

File: elements/a11y-media-player/src/a11y-media-player.js [hunk 2]
```diff
@@ -1498,29 +1499,16 @@ class A11yMediaPlayer extends SimpleColors {
...
-              end = a.endTime - b.endTime;
-            return start !== 0 ? start : end !== 0 ? end : a.track - b.track;
-          });
-        }
-      });
-    };
+    this.loadedTracks.textTracks.forEach(track => this._onAddTrack(track));
+    this.loadedTracks.textTracks.onremovetrack = e => this._onRemoveTrack(e.track);
+    this.loadedTracks.textTracks.onaddtrack = e => this._onAddTrack(e.track);
+    console.log('_addSourcesAndTracks',media,this.loadedTracks.textTracks,this.cues);
```
Comment (btopro) [inline elements/a11y-media-player/src/a11y-media-player.js]: boooooo, get out of here console.log, no body wants you

;)

File: elements/a11y-media-player/a11y-media-player.js [hunk 3]
```diff
@@ -2890,29 +2890,16 @@ ${this.poster
...
-            return start !== 0 ? start : end !== 0 ? end : a.track - b.track;
-          });
-        }
-      });
-    };
+    this.loadedTracks.textTracks.forEach(track => this._onAddTrack(track));
+    this.loadedTracks.textTracks.onremovetrack = e => this._onRemoveTrack(e.track);
+    this.loadedTracks.textTracks.onaddtrack = e => this._onAddTrack(e.track);
+    console.log('_addSourcesAndTracks',media,this.loadedTracks.textTracks,this.cues);
+
```
Comment (btopro) [inline elements/a11y-media-player/a11y-media-player.js]: console.log.

**Comment by nikkimk** [comment]
simple-modal fix

**Comment by nikkimk** [comment]
Handles how hax-preview deals with read-only/getters in lit

**Pull request merged**